scripts/functions.sh

```diff
@@ -267,6 +271,13 @@ function check_args_and_apply() {
         key="${option%%=*}"
         value="${option#*=}"
         case "$key" in
+            --copy) # Copy or skip without prompt
+                if [[ "$value" == "true" || "$value" == "false" || "$value" == "prompt" ]]; then
+                  copy="$value"
+                else
+                  echo "invalid value for --copy"
+                fi
+                ;;
```

Comment:
pls update doc with the new option

Reply:
https://github.com/netobserv/network-observability-cli/pull/65/commits/c5139edefb4064ec20cf0eb6aec67293ef8a6944